pmm_qa/pmm-framework.py

```diff
@@ -30,31 +31,24 @@
     },
 }
 
-def run_ansible_playbook(playbook_filename, env_vars, args):
-    # Install Ansible
-    try:
-        subprocess.run(['sudo', 'yum', 'install', 'ansible', '-y'])
-    except Exception as e:
-        print(f"Error installing Ansible: {e}")
-        exit(1)
 
+def run_ansible_playbook(playbook_filename, env_vars, args):
     # Get Script Dir
     script_path = os.path.abspath(sys.argv[0])
     script_dir = os.path.dirname(script_path)
     playbook_path = script_dir + "/" + playbook_filename
 
-    # Build the commands to execute the playbook
-    command = ["ansible-playbook", f"{playbook_path}", f'-e os_type=linux', f'--connection=local',
-               f'-l localhost', f'-i localhost,']
-
-    if args.verbose:
-        print(f'Options set after considering defaults: {env_vars}')
```

Comment:
Please restore this, required for --v,
its required as we have different ways to set ENV options, and helpful to debug.